src/eggd_sompy.sh

```
@@ -31,31 +51,35 @@ main() {
 
             prefix=$(basename $(basename ${query_vcf%%_*}))
 
+            bgzipped_query_vcf=$(bgzip_vcf $query_vcf $query_vcf_name)
+
             # normalise query VCF
             ## indexing fixes an issue seen with some inputs: "Contig '1' is not defined in the header. (Quick workaround: index the file with tabix.)"
-            tabix $query_vcf
-            query_vcf_name=$(sed -E 's/\.vcf(\.gz)?//g' <<< $(basename $query_vcf))
+            tabix $bgzipped_query_vcf
+            query_vcf_name=$(sed -E 's/\.vcf(\.gz)?//g' <<< $(basename $bgzipped_query_vcf))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)